lib/screens/test_screen/test_screen.dart

```diff
@@ -6,10 +7,45 @@ class TestScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: SingleChildScrollView(
-        child: Center(
-          child: CommonText(
-            text: 'my_profile'.tr,
+      body: SafeArea(
+        child: SingleChildScrollView(
+          child: Center(
+            child: Column(
+              children: <Widget>[
+                SizedBox(height: DEVICE_HEIGHT * 0.1),
+                CommonButton(
+                  buttonStyle: DefaultButtonStyles().defaultSmallButton(),
+                  text: 'Login'.tr,
+                ),
+                SizedBox(height: DEVICE_HEIGHT * 0.02),
+                Padding(
+                  padding:
+                      EdgeInsets.symmetric(horizontal: DEVICE_WIDTH * 0.08),
+                  child: CommonButton(
+                    buttonStyle: DefaultButtonStyles().defaultButtonStyle(),
```

Comment:
DefaultButtonStyles() is already created in my_app file, so  we need to make sure that we use the same instance we already created in my_app